### ISSUE TITLE
Supported Device not working (Grid CoOnnect Double)

### ISSUE THREAD

**Issue by jstride** [make-all/tuya-local#226]
I have the outdoor Grid Connect double outlet with Energy Monitoring, Master and Individual switches and Child Lock''. I can add the device via the GUI but once added I don't see the individual switches for each outlet (only the master), and no child lock. Is there something special I need to do - I simple added as switch and allocated the power, consumption and voltage (no option to define switches).

![image](https://user-images.githubusercontent.com/1501958/187348119-cd52ede5-b52a-4f83-a6e1-ea333ac9a2e4.png)

**Comment by jstride** [comment]
This is not an issue, for anyone adding this in the future you need to keep adding devices for each switch and they added as children.

**Comment by make-all** [comment]
This switch does not seem to have a config.  There is a Grid Connect double switch, but it is just the two switches with a few unknown attributes (not enough integer ones to be energy monitoring).

Which device was this detected as?

**Comment by jstride** [comment]
It is this one https://www.bunnings.com.au/deta-grid-connect-smart-outdoor-double-powerpoint_p0172781

I have manually added the two switches (in this screen shot, I have added a switch as x helper, hence the hidden Garden Lights) and the 'Pool Equipment' is the master switch. There is a separate child lock I have not yet added. If you let me know how to get the ID I can map the value if that would help?

![image](https://user-images.githubusercontent.com/1501958/187449834-cf1cf532-e6a1-48ee-ab11-365ac0c1a227.png)

![image](https://user-images.githubusercontent.com/1501958/187450061-4cfa75cf-8f4a-4299-9fe3-5949d2a0c87c.png)
